src/app/app.component.html

```diff
@@ -49,15 +49,15 @@ <h3 class="header">Technology</h3>
         </p>
         <h4 class="header">OSM Quality Analyses are powered by:</h4>
         <div class=""><a href="https://oqt.ohsome.org" target="_blank">
-          <img class="ui image" src="assets/images/ohsome-quality-analyst_without_fonts.svg">
+          <img class="ui image" src="assets/images/ohsome-quality-analyst_without_fonts.svg" alt="ohsome quality analyst logo">
```

Comment:
```suggestion
          <img class="ui image" src="assets/images/ohsome-quality-analyst_without_fonts.svg" alt="ohsome quality API logo">
```
ideally change the svg name as well

Reply:
@joker234 what about the Logo itself at the bottom: it shows "ohsome quality analyst". Should we remove it and only write ohsome quality API?

Reply:
same issue as https://github.com/GIScience/sketch-map-tool/issues/446

not relevant for this PR